examples/basic/vite.config.ts

```diff
@@ -8,5 +8,7 @@ export default defineConfig({
   test: {
     /* for example, use global to avoid globals imports (describe, test, expect): */
     // globals: true,
+    reporter: ['verbose'],
+    silentSkip: true,
```

Comment:
What about `hideSkippedTests`? I think this should also be available as a CLI flag